code/src/components/Wrapper.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+
+import ThoughtForm from './ThoughtForm';
+import ThoughtItems from './ThoughtItems';
+
+import { API_URL, LIKES_URL } from 'utils/urls';
```

Comment:
Very nice approach to create a folder with a component declaring the URL's there.